### PR TITLE
Added binary exponentiaion with respect to modulo

### PR DESCRIPTION
**Pull request by Anzoteh96**

### CODE REVIEW

File: maths/miller_rabin.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from binary_exp_mod import bin_exp_mod
```
Comment (cclauss) [inline maths/miller_rabin.py]: https://travis-ci.com/TheAlgorithms/Python/builds/133302996#L347

Try adding a . like `from .binary_exp_mod import bin_exp_mod`

File: maths/miller_rabin.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import random
+
+# This is a probabilistic check to test primality, useful for big numbers!
+def is_prime(n, prec = 1000):
+  if (n < 2):
```
Comment (cclauss) [inline maths/miller_rabin.py]: Please add one or more doctests here as discussed in CONTRIBUTING.md.

File: maths/miller_rabin.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+if __name__ == '__main__':
+  n = int(input("Enter bound : ").strip())
+  primes = []
+  for i in range(n + 1):
+    if is_prime(i):
+      primes.append(i)
+  print("Here's the list of primes")
+  for p in primes:
+    print(p, end=" ")
+  print()
```
Comment (cclauss) [inline maths/miller_rabin.py]:
```python
if __name__ == '__main__':
    n = abs(int(input("Enter bound : ").strip()))
    print("Here's the list of primes:")
    print(", ".join(str(i) for i in range(n + 1) if is_prime(i)))
```

File: maths/miller_rabin.py (hunk 2 above)
Comment (Anzoteh96) [reply maths/miller_rabin.py]: Added a test on it

File: maths/miller_rabin.py (hunk 3 above)
Comment (Anzoteh96) [reply maths/miller_rabin.py]: Did the change

**Comment by cclauss** [comment]
This file is not being tested https://travis-ci.com/TheAlgorithms/Python/builds/133415919#L417

**Pull request merged**